### DISCUSSION TITLE
希望添加删除视频页面鼠标指向UP头像和ID时弹出的元素功能

### DISCUSSION THREAD

**Discussion by camel5399-Git** [the1812/Bilibili-Evolved#2154] [功能建议 / Ideas]
完全不理解B站做的这个弹出到底有什么作用， 新窗口显示的东西全部在本体存在， 功能完全重复， 只会在我想点进UP空间看其他视频时弹出来挡我一下。

**Comment by Coulomb-G** [comment]
这些小改动你其实完全可以自己写,推荐一下「stylus」

**Reply by camel5399-Git** [reply]
然而我并不会写 😅

**Reply by Coulomb-G** [reply]
.user-card-m {
    display:none
}

**Reply by camel5399-Git** [reply]
😂看不懂，也不知道怎么用，我可以进行捐赠，能否添加功能

**Reply by the1812** [reply]
浏览器控制台(Ctrl + Shift + I)里运行这个:
```js
bilibiliEvolved.installStyle({ name: 'hideProfilePopup', displayName: '隐藏用户信息弹窗', style: `
.user-card-m {
  display: none !important;
}
`})
```

**Comment by Geeyun-JY3** [comment]
这种悬停弹出简易信息弹窗挺多网站都有的啊，比如 GitHub、Twitter、微博、知乎，可以让用户无需跳转就能直接在当前页面看到一些基本的信息。你可能没这种需求。

**Reply by camel5399-Git** [reply]
可是所谓的弹窗“无需跳转就能看到的基本信息”全都在原页面全部显示了，弹个窗出来再显示一边的意义在哪？

**Reply by camel5399-Git** [reply]
https://sm.ms/image/6s9mCzWR1QnPXOU
https://sm.ms/image/gifI1Ez5RDlFk7q

**Reply by Geeyun-JY3** [reply]
> 可是所谓的弹窗“无需跳转就能看到的基本信息”全都在原页面全部显示了，弹个窗出来再显示一边的意义在哪？

原页面<del>没等级（搞错了，是开了`简化评论区`才没），</del>没性别，也没个人空间 banner 啊，还能在弹窗看到无挂件的头像。还是看你需求。

**Reply by Geeyun-JY3** [reply]
好的现在 B 站把这玩意给增强了，显示了更多基本信息：多了认证信息、获赞数、具体是哪种大会员套餐
![image](https://user-images.githubusercontent.com/49396421/134758825-9224257e-a738-4670-9855-2b0fd6143b45.png)
